Add doc improvements related to environment and IdP configurations for Dev Portal (#1793)
wso2/docs-choreo-dev: en/developer-docs/docs/administer/configure-identity-providers-for-developer-portal.md

# Configure Identity Providers for Developer Portal

> You can manage which Identity Providers (IdPs) are available for consumers to authenticate and generate API keys in the Developer Portal.

Reviewer: ```suggestion
You can configure the Identity Providers (IdPs) available for consumers to authenticate and generate API keys in the Developer Portal from the Choreo Console.
```
